certified-connectors/document-ai-konfuzio/apiDefinition.swagger.json

```diff
@@ -0,0 +1,623 @@
+            "name": "sync",
+            "type": "boolean"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "File sent to Host including the URL to retrieve the extracted results",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data_file": {
+                  "title": "Data file",
+                  "type": "string",
+                  "readOnly": true,
+                  "format": "uri"
+                },
+                "id": {
+                  "title": "ID",
+                  "type": "integer",
+                  "readOnly": true
+                },
+                "project": {
+                  "title": "Project",
+                  "type": "integer"
+                },
+                "data_file_name": {
+                  "title": "Data file name",
+                  "type": "string",
+                  "readOnly": true,
+                  "minLength": 1
+                },
+                "callback_url": {
+                  "title": "Callback URL",
+                  "type": "string",
+                  "format": "uri",
+                  "maxLength": 200,
+                  "x-nullable": true
+                },
+                "sync": {
+                  "title": "Sync",
+                  "type": "boolean"
+                },
+                "extraction_url": {
+                  "title": "Extraction url",
+                  "type": "string",
+                  "readOnly": true
+                }
+              }
+            }
+          }
+        },
+        "tags": [
+          "docs"
+        ]
+      }
+    },
+    "/v2/docs/{doc}/": {
+      "get": {
+        "parameters": [
+          {
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Document ID",
+            "description": "ID of your document in the project",
+            "name": "doc",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "operationId": "v2_docs_read",
+        "summary": "Retrieve the extraction results for a document",
+        "description": "Get all information for a document using its id. The extraction results are available once the processing has finished",
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        },
+        "tags": [
+          "docs"
+        ]
+      },
+      "delete": {
+        "operationId": "v2_docs_delete",
+        "description": "Delete a document.",
+        "parameters": [
+          {
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Document ID",
+            "description": "ID of the document you want to delete",
+            "name": "doc",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": ""
```

Comment:
@atraining same here

Reply:
@sriyen-msft thanks for providing additional feedback. I have updated the response description.